README.md

```diff
@@ -248,6 +248,18 @@ As an example we use the Symfony `DateTimeNormalizer` service so we do have supp
 
 As a side note: If you happen to use [Autowiring](https://symfony.com/doc/current/service_container/autowiring.html) in your `services.yaml` you might need to set `autowire: false` too.
 
+**When namespace of Model used change**
+
+You must have migrated the data in the database because your JSON has `#type` key which contains the namespace of your used Model class.
+
+For example if we have a project that we migrate to Symfony/flex,
+we will have as namespace `AppBunle/Entity/Bar` for our Model class used who will become `App/Entity/Bar`.
+
+With MySQL we will have to do this:
+```
+UPDATE Baz SET misc = JSON_REPLACE(misc, '$."#type"', 'App\\\Entity\\\Bar');
```

Comment:
I don't like this example. It will replace everything, not just what was `AppBundle\Entity\Bar`. So this is not the correct query.